assets/javascripts/discourse/components/discourse-video-upload-form.js

```diff
@@ -117,15 +139,69 @@ export default Component.extend({
     ).test(fileName);
   },
 
+  isDurationAllowed() {
+    if (this.videoDurationMinutes === null) {
+      return false;
+    }
+    if (this.currentUser.staff) {
+      return true;
+    }
```

Comment:
```suggestion
    if (this.currentUser.staff) {
      return true;
    }
    if (this.videoDurationMinutes === null) {
      return false;
    }
```

I would flip these two around, so that staff can post videos even if, for some reason, we can't get the duration of the video.